posthog/tasks/test/test_email.py

```diff
@@ -98,6 +98,27 @@ def test_send_fatal_plugin_error(self, MockEmailMessage: MagicMock) -> None:
         assert mocked_email_messages[0].send.call_count == 1
         assert mocked_email_messages[0].html_body
 
+    def test_send_fatal_plugin_error_with_settings(self, MockEmailMessage: MagicMock) -> None:
+        mocked_email_messages = mock_email_messages(MockEmailMessage)
+        plugin = Plugin.objects.create(organization=self.organization)
+        plugin_config = PluginConfig.objects.create(plugin=plugin, team=self.team, enabled=True, order=1)
+        self._create_user("test2@posthog.com")
+        membership = self.user.organization_memberships.get()
```

Comment:
non-blocking

Similarly to above, you might also use the API to set the test conditions up, although I understand that that would be inconsistent with the other tests here.